web/backend/v2/src/test/java/com/tourneygen/web/WebApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.tourneygen.web;
```

Comment:
The filenames here are getting really long. Can we cut this down to web/backend/v2/com/.. ? or at least reduce the number of seemingly redundant directories here?

Reply:
this was all generated by the framework. Messing with it breaks gradle lets just leave it until it causes a problem. Dirs easy to fix